### PR TITLE
Add layout bg-messagease-phonetic-symbols

### PR DESCRIPTION
**Pull request by pavlov-p**
Add a layout for bulgarian language, stemming from MessagEase layout. Phonetic – mimicking the phonetic approach for MessagEase russian phonetic layout.

### CODE REVIEW

**Comment by pavlov-p** [comment]
Could you point me to what's failing the linting?

File: app/src/main/java/com/dessalines/thumbkey/keyboards/BGMessageEasePhoneticSymbols.kt [hunk 1]
```diff
@@ -0,0 +1,937 @@
...
+            listOf(
+                SPACEBAR_KEY_ITEM,
+                RETURN_KEY_ITEM,
+            ),
+        ),
+    )
+
+val KB_BG_MESSAGEASE_PHONETIC_SYMBOLS: KeyboardDefinition =
+    KeyboardDefinition(
+        title = "bulgarian symbols messageease phonetic",
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/BGMessageEasePhoneticSymbols.kt:926]: Check the other names. messageease should be at the very end.

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt [hunk 2]
```diff
@@ -94,94 +94,96 @@ import com.dessalines.thumbkey.keyboards.KB_UK_THUMBKEY
...
+    ENQwertyFour(82, KB_EN_QWERTYFOUR),
+    KZThumbKey(83, KB_KZ_THUMBKEY),
+    ENMIThumbKey(84, KB_EN_MI_THUMBKEY),
+    ENMIThumbKeySymbols(85, KB_EN_MI_THUMBKEY_SYMBOLS),
+    ESCAMessageEase(86, KB_ES_CA_MESSAGEEASE),
+    SVMessageEase(87, KB_SV_MESSAGEEASE),
+    ENDoubleSymbols(88, KB_EN_DOUBLE_SYMBOLS),
+    FIMessageEase(89, KB_FI_MESSAGEEASE),
+    ENDAThumbKeySymbols(90, KB_EN_DA_THUMBKEY_SYMBOLS),
+    ENThumbKey(91, KB_BG_MESSAGEASE_PHONETIC_SYMBOLS_SHIFTED)
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:188]: What's going on here? why did all these get re-added and changed indexes?

**Review by dessalines — CHANGES_REQUESTED** [review]

**Comment by dessalines** [comment]
You need to run `./gradlew formatKotlin` to pass lint.

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt (hunk 2 above)
Comment (pavlov-p) [reply app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:188]: My bad, initially I thought I should add the new one first, as it's first alphabetically. Then read the guide that new layouts should be added last. So I did fix/revert that change but it seems I didn't push the commit.

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt]: No probs.

**Pull request closed without merging**